scripts/kind-e2e/e2e.sh

```diff
@@ -258,7 +258,7 @@ for i in 2 3; do
                         --nattport ${ce_ipsec_nattport} \
                         --ikeport ${ce_ipsec_ikeport} \
                         --colorcodes ${subm_colorcodes} \
-			--disable-nat \
+			                  --disable-nat \
```

Comment:
If you’re going to fix this, please use spaces throughout; there are still tabs at the start of the line ;-).

Reply:
ack, thanks